package org.schabi.newpipe.about

/**
 * Class containing information about standard software licenses.
 */
object StandardLicenses {
    @JvmField
    val GPL3 = License("GNU General Public License, Version 3.0", "GPLv3", "gpl_3.html")

    @JvmField
    val APACHE2 = License("Apache License, Version 2.0", "ALv2", "apache2.html")

    @JvmField
    val MPL2 = License("Mozilla Public License, Version 2.0", "MPL 2.0", "mpl2.html")

    @JvmField
    val MIT = License("MIT License", "MIT", "mit.html")

    @JvmField
    val MIT0 = License("MIT No Attribution License", "MIT-0", "mit_0.html")

    @JvmField
    val EPL1 = License("Eclipse Public License, Version 1.0", "EPL 1.0", "epl1.html")

    @JvmField
    val BSD2 = License("BSD 2-Clause License", "BSD-2-Clause", "bsd_2_clause.html")

    @JvmField
    val BSD3 = License("BSD 3-Clause License", "BSD-3-Clause", "bsd_3_clause.html")
}
